abataub/rentroll
# Cypress save new record test and tests for TaskListDefinition & TaskList

Added UI tests for TaskListDefinitions:
1. Change Business To REX
2. Test on Grid Records
3. Test on Record Details Form (form opened after clicking on grid record)
4. Check default value of new record form.

Added UI tests for TaskListDefinitions:
1. Change Business To REX
2. Test on Grid Records
3. Test on Record Details Form (form opened after clicking on grid record)
4. Check default value of new record form.
